schema-bridge.graphql

```diff
@@ -426,11 +758,271 @@ type PoolHourlySnapshot @entity {
   rewardTokenEmissionsUSD: [BigDecimal!]
 }
 
+type PoolRouteSnapshot @entity {
+  " {PoolRoute ID}-{poolSnaphotID} "
+  id: ID!
+
+  " PoolRoute this snapshot belongs to "
+  poolRoute: PoolRoute!
+
+  " Timestamp of this snapshot "
+  timestamp: BigInt!
+
+  " Block number of this snapshot "
+  blockNumber: BigInt!
+
+  " volume received through this route during the time of the snapshot period (hourly/daily) "
+  snapshotVolumeIn: BigInt!
+
+  " total volume received through this route in inputToken "
+  cumulativeVolumeIn: BigInt!
+
+  " volume sent through this route during the time of the snapshot period (hourly/daily) "
+  snapshotVolumeOut: BigInt!
+
+  " total volume sent through this route in inputToken "
+  cumulativeVolumeOut: BigInt!
+
+  " volume received through this route during the time of the snapshot period (hourly/daily) in USD "
+  snapshotVolumeInUSD: BigDecimal!
+
+  " total volume received through this route in USD "
+  cumulativeVolumeInUSD: BigDecimal!
+
+  " volume sent through this route during the time of the snapshot period (hourly/daily) in USD "
+  snapshotVolumeOutUSD: BigDecimal!
+
+  " total volume sent through this route in USD "
+  cumulativeVolumeOutUSD: BigDecimal!
+}
+
+##################################
+##### Transaction-Level Data #####
+##################################
+
+"""
+An event is any user action that occurs in a protocol. Generally, they are Ethereum events
+emitted by a function in the smart contracts, stored in transaction receipts as event logs.
+However, some user actions of interest are function calls that don't emit events. For example,
+the deposit and withdraw functions in Yearn do not emit any events. In our subgraphs, we still
+store them as events, although they are not technically Ethereum events emitted by smart
+contracts.
+"""
+interface Event {
+  " { Event type }-{ Transaction hash }-{ Log index } "
+  id: ID!
+
+  " Transaction hash of the transaction that emitted this event "
+  hash: String!
+
+  " Event log index. For transactions that don't emit event, create arbitrary index starting from 0 "
+  logIndex: Int!
+
+  " The protocol this transaction belongs to "
+  protocol: BridgeProtocol!
+
+  " Address that received the tokens "
+  to: String!
+
+  " Address that sent the tokens "
+  from: String!
+
+  " Block number of this event "
+  blockNumber: BigInt!
+
+  " Timestamp of this event "
+  timestamp: BigInt!
+}
+
+type LiquidityDeposit implements Event @entity {
```

Comment:
I think `LiquidityDeposit`, `LiquidityWithdraw`, `BridgeTransfer` events should also have an `amount` field

Reply:
I’m not sure how I missed that 😅

Reply:
Added those